ansible/roles/ansible-horizon-plugin/tasks/install.yml

```diff
@@ -8,7 +8,7 @@
        
 - name: install pip if it is not installed 
   shell: easy_install --no-deps http://{{IP_ADDRESS}}:{{PORT_NO}}/packages/pip-{{pip_version}}.tar.gz 
-  register: pip_installation
+  ignore_errors: yes
```

Comment:
Not sure if we should ignore this. pip has to be installed as subsequent steps require it to install different packages.